wpilibNewCommands/src/main/native/include/frc2/command/Command.h

```diff
@@ -12,6 +12,7 @@
 #include <units/time.h>
 #include <wpi/Demangle.h>
 #include <wpi/SmallSet.h>
+#include <wpi/deprecated.h>
```

Comment:
Why is this header added?

Reply:
Command.h uses the WPI_DEPRECATED() from that header, so it's being included to adhere to include-what-you-use.